backend/openapi.yaml

```diff
@@ -4421,13 +4439,36 @@ components:
           $ref: '#/components/schemas/GovSGTemplate'
     GovSGTemplate:
       type: object
+      required:
+        - id
+        - body
+        - params
       properties:
+        id:
+          type: number
         body:
           type: string
         params:
           type: array
           items:
             type: string
+        multilingual_support:
```

Comment:
don't really understand how this part works, but I'm guessing our own template actually abstracts across multiple WA's templates?

Reply:
Yup so basically it's an array that contains information about the template in alternative languages, which only differ in: 
- `language_code` (e.g. `en_GB`, `zh_CN`, etc)
- `language`: This is mainly for frontend display purposes (e.g. `English (UK)`, `Chinese (CN)`, etc)
- `body`: template body in the alternative language